### ISSUE TITLE
link to pro broken

### ISSUE THREAD

**Issue by kate-codec** [pluginever/wp-content-pilot#194]
hi please i'vee tried several time to purchas the pro version but all the links i clicked responded with error "Error 404.
Bummer. It looks like this link is broken."

**Comment by wiredafrican** [comment]
Yes, unfortunately I think this plugin has been abandoned. I bought the Pro version in 2020 and it's no longer working as a pro version plugin. I was hoping we could get someone to add ChatGPT into the plugin. :-(
